.gitignore

```diff
@@ -11,3 +11,4 @@
 
 # Output of the go coverage tool, specifically when used with LiteIDE
 *.out
+coverage.txt
```

Comment:
Nit: this is called `coverage.out` in most places (like https://blog.golang.org/cover)